### PR TITLE
Fixed some grammers in the number guessing game

### PR DESCRIPTION
**Pull request by balogun14**
I fixed some Crammers in the guessing game main.py

### CODE REVIEW

File: Number Guessing Game/main.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def check_win(player_guess):
     attempts += 1
 
     if check_win(guess):
-        print('You won! - Number of attempts: ' + str(attempts))
+        print('You won! with  - Number of attempts: ' + str(attempts))
```
Comment (AJThePro99) [inline Number Guessing Game/main.py:34]: Original message is more straight to the point and clean.
'with' also creates breaks the flow of the sentence, especially when you are putting a hyphen in between to separate the two sentences.

**Review by AJThePro99 — CHANGES_REQUESTED** [review]
Line 34 needs to be reverted.
Original sentence is more concise and straightforward.

File: Number Guessing Game/main.py (hunk 1 above)
Comment (balogun14) [reply Number Guessing Game/main.py:34]: okay i understand sir

**Comment by balogun14** [comment]
i forgot to close the pr

**Pull request closed without merging**